addon/components/sortable-th.js

```diff
@@ -11,6 +11,9 @@ export default class SortableTh extends Component {
   }
 
   get sortedAscending() {
+    if (false === this.args.sortedAscending) {
+      return this.args.sortedAscending;
+    }
     return this.args.sortedAscending || true;
```

Comment:
I think this was supposed to be `return this.args.sortedAscending ?? true;` which would return the value if set or else return true and should work without the addition to check for explicit false.

Reply:
good point. replaced this with the suggested shorthand form.